src/components/Contentful/ContentfulDynamicHero.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+	<section class="contentful-hero">
+		<div class="row align-center">
+			<!-- hero media -->
+			<div
+				class="small-12 medium-10 large-6 xlarge-5 columns"
+			>
+				<template v-if="isHeroImage">
+					<router-link
+						:to="heroButton.link"
+						v-kv-track-event="[
+							'homepage',
```

Comment:
we'll need to dynamically derive these contexts...it may be helpful to have a util we can import that contains a map of urls to values we want to use in analytics events so we can make sure to match our event standards.

_There are some other contexts that might be better served with a prop of analytics data..._